src/backend/db/products.js

```diff
@@ -12,252 +12,53 @@ export const products = [
-    categoryName: "Kids",
+    subCategory: "Shoes",
+    rating: "3"
   },
   {
     _id: uuid(),
     title: "Nike Air Force 1 '07",
     price: "5000",
-    image: "https://static.nike.com/a/images/t_PDP_1728_v1/f_auto,b_rgb:f5f5f5/b7d9211c-26e7-431a-ac24-b0540fb3c00f/air-force-1-07-shoe-WrLlWX.png",
-    categoryName: "Men",
-  },
-  {
-    _id: uuid(),
-    title: "Nike Air Max 90 SE",
-    price: "3000",
-    image: "https://static.nike.com/a/images/t_PDP_1728_v1/f_auto,b_rgb:f5f5f5/71b76b15-f9e4-4a0e-9654-a9f8dc373597/air-max-90-se-shoes-lxzG1b.png",
+    image: "https://static.nike.com/a/images/t_PDP_1728_v1/f_auto,b_rgb:f5f5f5/58e6ba49-6035-4cac-9907-32e7de72df83/dri-fit-one-mid-rise-leggings-9ct6Z3.png",
     categoryName: "Women",
-  },
-  {
-    _id: uuid(),
-    title: "Nike3",
-    price: "1000",
-    image: "https://static.nike.com/a/images/t_PDP_1728_v1/f_auto,b_rgb:f5f5f5/b7d9211c-26e7-431a-ac24-b0540fb3c00f/air-force-1-07-shoe-WrLlWX.png",
-    categoryName: "Kids",
-  },
-  {
-    _id: uuid(),
-    title: "Nike Air Force 1 '07",
-    price: "5000",
-    image: "https://static.nike.com/a/images/t_PDP_1728_v1/f_auto,b_rgb:f5f5f5/b7d9211c-26e7-431a-ac24-b0540fb3c00f/air-force-1-07-shoe-WrLlWX.png",
-    categoryName: "Men",
+    subCategory: "Clothing",
+    rating: "4"
   },
   {
     _id: uuid(),
     title: "Nike Air Max 90 SE",
     price: "3000",
-    image: "https://static.nike.com/a/images/t_PDP_1728_v1/f_auto,b_rgb:f5f5f5/71b76b15-f9e4-4a0e-9654-a9f8dc373597/air-max-90-se-shoes-lxzG1b.png",
-    categoryName: "Women",
-  },
-  {
-    _id: uuid(),
-    title: "Nike3",
-    price: "1000",
-    image: "https://static.nike.com/a/images/t_PDP_1728_v1/f_auto,b_rgb:f5f5f5/b7d9211c-26e7-431a-ac24-b0540fb3c00f/air-force-1-07-shoe-WrLlWX.png",
+    image: "https://static.nike.com/a/images/t_PDP_1728_v1/f_auto,b_rgb:f5f5f5/4c629df2-9840-4982-aa92-918f63d51a8b/flex-advance-younger-shoes-rXcKzS.png",
     categoryName: "Kids",
-  },
-  {
-    _id: uuid(),
-    title: "Nike Air Force 1 '07",
-    price: "5000",
-    image: "https://static.nike.com/a/images/t_PDP_1728_v1/f_auto,b_rgb:f5f5f5/b7d9211c-26e7-431a-ac24-b0540fb3c00f/air-force-1-07-shoe-WrLlWX.png",
-    categoryName: "Men",
-  },
-  {
-    _id: uuid(),
-    title: "Nike Air Max 90 SE",
-    price: "3000",
-    image: "https://static.nike.com/a/images/t_PDP_1728_v1/f_auto,b_rgb:f5f5f5/71b76b15-f9e4-4a0e-9654-a9f8dc373597/air-max-90-se-shoes-lxzG1b.png",
-    categoryName: "Women",
+    subCategory: "Shoes",
+    rating: "5"
   },
   {
     _id: uuid(),
     title: "Nike3",
     price: "1000",
-    image: "https://static.nike.com/a/images/t_PDP_1728_v1/f_auto,b_rgb:f5f5f5/b7d9211c-26e7-431a-ac24-b0540fb3c00f/air-force-1-07-shoe-WrLlWX.png",
+    image: "https://static.nike.com/a/images/t_PDP_1728_v1/f_auto,b_rgb:f5f5f5/3e797e14-9abe-44b7-85c7-d25f5751a2d3/pro-older-short-sleeve-training-top-1z0zqt.png",
```

Comment:
Suggestion: You can store all the images in one object and then extract the image url in image property

example:
const imgObj = {
    nike3: "/image/url/"
}

you could it as:
image: imgObj.nike3